app/models/lead_partner.rb

```diff
@@ -40,6 +40,9 @@ class LeadPartner < ApplicationRecord
   has_many :lead_partner_users, inverse_of: :lead_partner
   has_many :users, through: :lead_partner_users
 
+  has_many :funding_payment_schedules, class_name: "Funding::PaymentSchedule", as: :payable
+  has_many :funding_trainee_summaries, class_name: "Funding::TraineeSummary", as: :payable
```

Comment:
I think for those 2 to work for any existing `schools` or `providers` we'll have to update [CopyLeadSchoolToLeadPartnerService](https://github.com/DFE-Digital/register-trainee-teachers/blob/7168-add-lead-partner-funding-in-support-console/app/services/copy_lead_school_to_lead_partner_service.rb#L3)